package.json

```diff
@@ -33,23 +34,37 @@
     "prepublishOnly": "npm run build && npm test"
   },
   "jest": {
-    "moduleFileExtensions": [
-      "ts",
-      "js"
-    ],
-    "coveragePathIgnorePatterns": [
-      "__tests__",
```

Comment:
The old code used `__tests__` and `__checks__` folders. Can we keep that structure or is that hard?

(If we're starting out with the hope that this fork goes away eventually, it might be nice to avoid unnecessary changes.)

Reply:
I moved this for less ignore configuration in these files, but if you'd prefer I can switch it back.

Reply:
Yeah, I think I'd prefer to leave it unchanged in this case, to stay close to the upstream code.

(I don't think that's _always_ the right answer, e.g. changing the type testing library like a change that's worth it.)